src/components/ContinuationIn/ContinuationAuthorization.vue

```diff
@@ -0,0 +1,393 @@
+<template>
+  <!--
+    Don't render this component until authorization object is initialized, so that we mount
+    FileUploadPreview with existing files, because its "inputFile" prop is not reactive.
+  -->
+  <div
+    v-if="authorization"
+    id="continuation-authorization"
+  >
+    <header>
+      <p>
+        Upload the authorization for the continuation from the foreign jursidiction.
+      </p>
+
+      <ul>
+        <li>
+          Use a white background and a legible font with contrasting font colour
+        </li>
+        <li>
+          PDF file type (maximum 30 MB file size)
+        </li>
+      </ul>
+    </header>
+
+    <div class="pt-4" />
+
+    <v-row
+      v-for="(num, index) in numUploads"
+      :key="authorization.files[index]?.fileKey"
+      class="mt-4 mb-n2"
+      no-gutters
+    >
+      <v-col
+        cols="12"
+        sm="3"
+      >
+        <label v-if="index === 0">Upload File (5 maximum)</label>
+      </v-col>
+
+      <v-col
+        cols="12"
+        sm="9"
+      >
+        <FileUploadPreview
+          inputFileLabel="Continuation authorization"
+          :maxSize="MAX_FILE_SIZE"
+          :pdfPageSize="PdfPageSize.LETTER_SIZE"
+          :inputFile="authorization.files[index]?.file"
+          :showErrors="getShowErrors"
+          :customErrorMessage="fileUploadCustomErrorMsg"
+          :isRequired="getShowErrors && (index === 0)"
+          @fileValidity="onFileValidity($event)"
+          @fileSelected="onFileSelected(index, $event)"
+        />
+      </v-col>
+    </v-row>
+
+    <v-form
+      ref="formRef"
+      lazy-validation
+      @submit.prevent
+    >
+      <v-row
+        class="mt-6"
+        no-gutters
+      >
+        <v-col
+          cols="12"
+          sm="3"
+        >
+          <label>Authority Name</label>
+        </v-col>
+
+        <v-col
+          cols="12"
+          sm="9"
+          class="pl-8"
+        >
+          <v-text-field
+            v-model="authorization.authorityName"
+            class="authority-name"
+            filled
+            hide-details="auto"
+            label="Authority Name"
+            :rules="getShowErrors ? authorityNameRules : []"
```

Comment:
This is a small trick to enable the rules only once the global "show errors" flag is set to True.